typescript/web/src/components/workspaces/update-plan-banner.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, chakra, Flex, Text } from "@chakra-ui/react";
+import { useRouter } from "next/router";
+import { RiInformationFill } from "react-icons/ri";
+import { useWorkspace } from "../../hooks/use-user";
+
+const InfoIcon = chakra(RiInformationFill);
+export const UpdatePlanBanner = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/2091902/155528892-3bca66d5-b722-4ff7-8bc6-4e6fe4c4c414.png)

IDK if we talked about it yet in DSM or other but you can install the CodeClimate Chrome extension to see such lines in GitHub diff 😉